src/Test/TestCases.Runtime/MissingRequiredExtensionTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿// This file is part of Core WF which is licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+using Shouldly;
+using System.Activities;
+using System.Activities.Statements;
+using System.Text;
+using Xunit;
+
+namespace TestCases.Runtime
+{
+    public class MissingRequiredExtensionTests
```

Comment:
Maybe we should add also a test to verify that no exception is thrown when a type is successfully resolved. I haven't seen one such test from what I searched.